### PR TITLE
LayerZero NFT

### PR DESCRIPTION
**Pull request by Kud8**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -154,6 +154,7 @@ dist
 cache
 artifacts
 submodule-artifacts/goerli/solcInputs
```
Comment (elpinguinofrio) [inline .gitignore:156]: why do we exclude solc inputs?

File: README.md [hunk 2]
```diff
@@ -265,3 +265,12 @@ npx hardhat transfer-owner --network mumbai --address 0x000000006cD799E2cC7A3Fe6
 4. In the proposal - addresses, that are in the gnosis-safe multisig, can approve this proposal
 5. After that you can execute it.
 6. If proposal is executed you can verify contract via `npm run verify:goerli`
+
+### CrossChain NFT
+
```
Comment (elpinguinofrio) [inline README.md:270]: please add link to their docs

File: README.md [hunk 3]
```diff
@@ -265,3 +265,12 @@ npx hardhat transfer-owner --network mumbai --address 0x000000006cD799E2cC7A3Fe6
...
+We use Layer-Zero UniversalONFT721 in the base. So for make NFT cross-chain sendable you should:
+1. Deploy to chain1 with `npm run deploy-nft:goerli`
+2. Deploy to chain2 with `npm run deploy-nft:mumbai`
+3. Prepare NFT in chain1 with `hardhat run ./scripts/prepare-lz-nft.ts --network mumbai`
+4. Prepare NFT in chain2 with `hardhat run ./scripts/prepare-lz-nft.ts --network goerli`
+5. For testing tha all works - run script that send token with id 100 from deployer in mumbai to admin in goerli with `hardhat run ./scripts/send-lz-nft.ts --network mumbai`
```
Comment (elpinguinofrio) [inline README.md:276]: *that, *sends

File: hardhat.config.ts [hunk 4]
```diff
@@ -102,7 +111,20 @@ const config: HardhatUserConfig = {
         currency: "USD",
     },
     etherscan: {
-        apiKey: ETHERSCAN_API_KEY,
+        apiKey: {
+            goerli: ETHERSCAN_API_KEY ?? "",
+            polygonMumbai: ETHERSCAN_API_KEY ?? "",
+        },
+        customChains: [
```
Comment (elpinguinofrio) [inline hardhat.config.ts:118]: please write in comments why do we need custom chain

File: scripts/send-lz-nft.ts [hunk 5]
```diff
@@ -0,0 +1,86 @@
+// We require the Hardhat Runtime Environment explicitly here. This is optional
+// but useful for running the script in a standalone fashion through `node <script>`.
+//
+// When running the script with `npx hardhat run <script>` you'll find the Hardhat
+// Runtime Environment's members available in the global scope.
+import { ethers, getNamedAccounts} from "hardhat";
```
Comment (elpinguinofrio) [inline scripts/send-lz-nft.ts:6]: it would be great, if you write a one-two sentences of what this script does, and why is needed. Same for prepare script

**Review by elpinguinofrio — CHANGES_REQUESTED** [review]
looks good overall! 
1) please make requested changes. 
2) I haven't tested bridge itself, will I able to do it myself? if it's take more than 5 minutes on my end, please record loom video demonstration how do you do it yourself.

File: .gitignore (hunk 1 above)
Comment (Kud8) [reply .gitignore:156]: because every new deploy it has a lot of changes, that we don't want to see in PR

File: hardhat.config.ts (hunk 4 above)
Comment (Kud8) [reply hardhat.config.ts]: we don't need that

**Comment by Kud8** [comment]
> looks good overall!
> 
>     1. please make requested changes.
> 
>     2. I haven't tested bridge itself, will I able to do it myself? if it's take more than 5 minutes on my end, please record loom video demonstration how do you do it yourself.

1) Done
2) You could do it in less than 5 minutes if you run all task as described in Readme. For example, you can run 1 and 2 steps in different consoles at the same time. (also you can run 3 and 4 steps in the same way).

**Pull request merged**